database/core_templates/schema_init.sql

```diff
@@ -1784,60 +1784,66 @@ CREATE TYPE public.reviewer_action AS ENUM (
     'CONTINUE_REVIEW',
     'MAKE_DECISION',
     'RESTART_REVIEW',
-    'UPDATE_REVIEW'
+    'UPDATE_REVIEW',
+    'AWAITING_RESPONSE'
 );
 
-CREATE FUNCTION review_list (stageid int, reviewerid int)
+CREATE FUNCTION review_list (stageid int, reviewerid int, appstatus public.application_status)
     RETURNS TABLE (
         application_id int,
         reviewer_action public.reviewer_action
     )
     AS $$
-    SELECT
-        review_assignment.application_id AS application_id,
-        CASE WHEN COUNT(*) FILTER (WHERE review_status_history.status = 'CHANGES_REQUESTED') != 0 THEN
-            'UPDATE_REVIEW'
-        WHEN COUNT(*) FILTER (WHERE review_status_history.status = 'PENDING') != 0 THEN
-            'RESTART_REVIEW'
-        WHEN COUNT(*) FILTER (WHERE review_status_history.status = 'DRAFT'
-            AND is_locked = FALSE) != 0 THEN
-            'CONTINUE_REVIEW'
-        WHEN COUNT(*) FILTER (WHERE review_assignment.status = 'ASSIGNED'
-            AND review_assignment.is_final_decision = TRUE
-            AND review_assignment.is_last_stage = TRUE
-            AND review = NULL) != 0 THEN
-            'MAKE_DECISION'
-        WHEN COUNT(*) FILTER (WHERE review_assignment.status = 'ASSIGNED'
-            AND review.id IS NULL) != 0 THEN
-            'START_REVIEW'
-        WHEN COUNT(*) FILTER (WHERE review_assignment.status = 'AVAILABLE'
```

Comment:
Unnecessary whitespace changes. I've fixed it in the main sql files, so this should be fixed when you re-export core_templates.